apps/dg/components/case_table/case_table_controller.js

```diff
@@ -161,7 +161,39 @@ DG.CaseTableController = DG.ComponentController.extend(
         this.updateTableAdapters();
 
         var contentView = this.getPath('view.contentView');
-        if( contentView) contentView.setCaseTableAdapters( this.get('caseTableAdapters'));
+
+        // Handle divider views for the drag-attribute listener setup.
+        if (contentView) {
+          var dividerDropDataCallback = function (iDivider) {
+            var dropData = iDivider.getPath('dropData');
+            if (SC.none(dropData)) {
+              return;
+            }
+            var context = iDivider.getPath('dataContext');
+            var parentTable = iDivider.getPath('leftTable');
+
+            // TODO: Add the rest of UNDO procedures in createCollectionCommand.
```

Comment:
Is this TODO complete? If so, maybe the comment should be removed.

Reply:
It has not been done yet. I could work on it as a separate story?